server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java

```diff
@@ -2432,13 +2437,24 @@ private PartialPath convertConstantToPath(String src) throws IllegalPathExceptio
 
   // node name
 
-  /** function for parsing node name. */
   private String parseNodeName(IoTDBSqlParser.NodeNameContext ctx) {
-    return ctx.getText();
+    return parseNodeString(ctx.getText());
   }
 
   private String parseNodeNameWithoutWildCard(IoTDBSqlParser.NodeNameWithoutWildcardContext ctx) {
-    return ctx.getText();
+    return parseNodeString(ctx.getText());
+  }
+
+  private String parseNodeString(String nodeName) {
```

Comment:
This method is duplicated.

Reply:
NodeName and NodeNameWithoutWildCard could be different in the future, maybe it is better this way?

Reply:
Got it, let's leave them separated.